apps/meteor/ee/app/livechat-enterprise/server/hooks/onCloseLivechat.ts

```diff
@@ -20,7 +21,11 @@ const onCloseLivechat = async (params: LivechatCloseCallbackParams) => {
 		LivechatRooms.unsetOnHoldByRoomId(roomId),
 		Subscriptions.unsetOnHoldByRoomId(roomId),
 		AutoCloseOnHoldScheduler.unscheduleRoom(roomId),
-	]);
+	]).then((data) => {
```

Comment:
pls don't use `then()` expect if really needed (like cannot convert a function to `async`)